backend/build.gradle

```diff
@@ -27,6 +27,9 @@ plugins {
     id "com.github.ben-manes.versions" version "0.20.0"
     id "com.diffplug.gradle.spotless" version "3.14.0"
     id "info.solidsoft.pitest" version "1.3.0"
+    id "com.moowork.node" version "0.11"
```

Comment:
As always: Have you checked the licences for the new dependencies? Are their terms permissive enough for us?

Reply:
All 3 are Apache-2.0 licensed, permitting reuse.

Reply:
[Source](https://github.com/srs/gradle-node-plugin)